src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDayCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let total = days * perDayCost;
+
+  if (days >= 3 && days < 7) {
+    total -= smallDiscount;
+  }
+
+  if (days >= 7) {
+    total -= bigDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    total -= bigDiscount;
  } else if (days >= 3) {
    total -= smallDiscount;
  }
```

You can switch these statements by place and get rid of one redundant check.
Also, use `if - else if` construction